components/employees/add-employee-form/index.tsx

```diff
@@ -0,0 +1,453 @@
+      >
+        <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 md:gap-6">
+          <FormField
+            control={form.control}
+            name={"firstName"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel required>{firstName?.label}</FormLabel>
+
+                <FormControl>
+                  <Input
+                    placeholder={firstName.placeholder}
+                    type={firstName.type}
+                    {...field}
+                  />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name={"lastName"}
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel required>{lastName?.label}</FormLabel>
+
+                <FormControl>
+                  <Input
+                    placeholder={lastName.placeholder}
+                    type={lastName?.type}
+                    {...field}
+                  />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name={"email"}
+            render={({ field }) => (
+              <FormItem className="sm:col-span-2">
+                <FormLabel required>{email?.label}</FormLabel>
+
+                <FormControl>
+                  <Input
+                    placeholder={email?.placeholder}
+                    type={email.type}
+                    {...field}
+                  />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <FormField
+            control={form.control}
+            name={"hireDate"}
+            render={({ field }) => {
+              return (
+                <FormItem>
+                  <FormLabel>{hireDate?.label}</FormLabel>
+
+                  <FormControl>
+                    <DatePicker
+                      onSelect={field.onChange}
+                      placeholder={probationEnd?.placeholder}
+                      value={field?.value}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              );
+            }}
+          />
+
+          <FormField
+            control={form.control}
+            name="role"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>{role?.label}</FormLabel>
+
+                <Select
+                  onValueChange={field.onChange}
+                  defaultValue={field.value}
+                >
+                  <FormControl>
+                    <SelectTrigger
+                      className={`${field?.value === undefined ? "text-placeholder" : "text-textColor"}`}
```

Comment:
use placeholder:text-placeholder instead

Reply:
@imrankhan-stixor , this is not an input field so `placeholder:text-placeholder` will not work